catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/NitfGmtiTransformer.java

```diff
@@ -150,7 +158,8 @@ private void transformAircraftLocation(Metacard metacard) {
                 WKTReader wktReader = new WKTReader(geometryFactory);
                 Geometry geometry = wktReader.read(aircraftLocation);
 
-                LOGGER.debug("Setting the metacard attribute [{}, {}]", Isr.DWELL_LOCATION,
+                LOGGER.debug("Setting the metacard attribute [{}, {}]",
```

Comment:
This should be `trace`. See https://codice.atlassian.net/wiki/display/DDF/Log+Levels.